### ISSUE TITLE
TextArea needs way to get text and to delete oldest lines

### ISSUE THREAD

**Issue by RolandHughes** [mitsuba-renderer/nanogui#95]
All,

I could just be blind, but I cannot see any way with TextArea to actually obtain the text or delete oldest lines programmatically. 

With other UI's when writing verification software you simply create a text area with a scroll bar and stream your output to it. Most offer a way of setting and getting the text (as well as appending) and a way of counting lines. Usually one sets a constant of 500-8000 for the scrollback limit. On the lower level UIs you have to count the lines and delete the oldest/front/top when you hit to scrollback limit.

I'm not seeing a way to do that with TextArea. If TextArea is in a VScrollPanel won't it just continue to grow until it exhausts all RAM?

Basically we are simulating terminal output. Touch screen, no keyboard or mouse. If there is something I can set with VScrollPanel that will let older text "just fall off the top" that would be cool too.
